src/app/home/home.component.html

```diff
@@ -3,105 +3,111 @@
 <body>
 
 <section class="vf-hero vf-hero--800 | vf-u-fullbleed home-background" >
-  <div class="vf-hero__content | vf-box | vf-stack vf-stack--400 home-summary">
-    <h2 class="vf-hero__heading pl-0">FAIR Wizard</h2>
-    <p class="vf-hero__subheading">FAIR principles serve to guide data producers and data publishers to improve their data management practices and improve the reuse of research data. FAIRification is the process of improving the FAIR level of datasets, projects or related infrastructure.</p>
-    <a class="vf-hero__link" href="/components">Learn more
-      <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg">
-        <path d="M0 12c0 6.627 5.373 12 12 12s12-5.373 12-12S18.627 0 12 0C5.376.008.008 5.376 0 12zm13.707-5.209l4.5
-        4.5a1 1 0 010 1.414l-4.5 4.5a1 1 0 01-1.414-1.414l2.366-2.367a.25.25 0 00-.177-.424H6a1 1 0 010-2h8.482a.25.25
-        0 00.177-.427l-2.366-2.368a1 1 0 011.414-1.414z" fill="" fill-rule="nonzero"></path>
-      </svg>
-    </a></div>
+<div class="vf-hero__content | vf-box | vf-stack vf-stack--400 home-summary">
+  <h2 class="vf-hero__heading pl-0">FAIR Wizard</h2>
+  <p class="vf-hero__subheading">The FAIR Wizard allows people without specific prior knowledge to explore solutions to help them adopt the  FAIR principles, as well as practical steps they can take to do so, supported by contextually-relevant documentation and tools. It also connects different FAIR communities and serves as a unified entrance for finding resources related to FAIR.</p>
+  <!-- TODO This link lead to no where. TODO where should we direct the users to -->
+  <a class="vf-hero__link" href="/components">Learn more
+    <svg width="24" height="24" xmlns="http://www.w3.org/2000/svg">
+      <path d="M0 12c0 6.627 5.373 12 12 12s12-5.373 12-12S18.627 0 12 0C5.376.008.008 5.376 0 12zm13.707-5.209l4.5
+      4.5a1 1 0 010 1.414l-4.5 4.5a1 1 0 01-1.414-1.414l2.366-2.367a.25.25 0 00-.177-.424H6a1 1 0 010-2h8.482a.25.25
+      0 00.177-.427l-2.366-2.368a1 1 0 011.414-1.414z" fill="" fill-rule="nonzero"></path>
+    </svg>
+  </a></div>
```

Comment:
Output is not bit out of box rendered. I can not say which lines changed exactly becasue of changes in indentation. Can you check.

Reply:
fixed by removing the "learn more" link. It doesn't go anyway anyway